packages/HilltopCrawler/src/main/kotlin/nz/govt/eop/hilltop_crawler/db/DB.kt

```diff
@@ -127,7 +127,7 @@ class DB(val template: JdbcTemplate, val objectMapper: ObjectMapper) {
         """
           UPDATE hilltop_fetch_tasks
           SET previous_data_hash = ?, 
-              previous_history = previous_history || ?::jsonb,    
+              previous_history = jsonb_path_query_array(previous_history, '$[last-49 to last]') || ?::jsonb,    
```

Comment:
This is a bit separate to the overall PR but this just makes us only keep the last 50 requests in history (we already had some where the history was in the 1000's)